nav2_collision_monitor/README.md

```diff
@@ -0,0 +1,61 @@
+# Nav2 Collision Monitor
+
+Collision Monitor - is an independent layer in Nav2 providing an additional level of robot safety assurance.
+It allows performing robot collision avoidance by monitoring the obstacles in surrounding environments laying in collision proximity to the robot.
+
+This is extremely useful and integral part of large heavy industrial robots operating in the facilities, or of robots moving with high velocities, where the collision with any obstacle might cause unpredictable severe consequences.
+
+## Features
+
+Collision Monitor operates by means of areas exposed around the robot and moving with it.
```

Comment:
The Collision Monitor uses polygons relative the robot's base frame origin to define "zones". Data that fall into these zones trigger an operation depending on the model being used. A given instance of the Collision Monitor can have many zones with different models at the same time. When multiple zones trigger at once, the most aggressive one is used (e.g. stop > slow 50% > slow 10%).

Reply:
Done in latest changes.